x-pack/legacy/plugins/epm/common/types.ts

```diff
@@ -108,28 +124,19 @@ export interface Dataset {
   type: string;
 }
 
-export interface RegistryPackage {
-  name: string;
-  version: string;
-  description: string;
-  readme?: string;
-  icon: string;
-  requirement: RequirementsByServiceName;
-  title?: string;
-  screenshots?: ScreenshotItem[];
-  assets: string[];
-  datasets: Dataset[];
+// some properties are optional in Registry responses but required in EPM
+// internal until we need them
+interface PackageAdditions {
+  title: string;
+  assets: AssetsGroupedByServiceByType;
 }
 
 // Managers public HTTP response types
 export type PackageList = PackageListItem[];
 
-export type PackageListItem = Installable<Required<RegistryListItem>>;
+export type PackageListItem = Installable<RegistryListItem & PackageAdditions>;
```

Comment:
Remove the `Required<RegistryListItem>` which said all items in `RegistryListItem` were required, which is not true. Any items which are optional in `RegistryListItem` are optional in `PackageListItem`